version/version_lib.go

```diff
@@ -0,0 +1,16 @@
+// +build !autogen
+
+// Package version is auto-generated at build-time
+package version
+
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GITCOMMIT string = "library-import"
```

Comment:
I'm pretty sure this specific constant predates the coding style, but a PR
to update this to conform would definitely be great IMO.